.github/workflows/spellcheck.yml

```
@@ -0,0 +1,39 @@
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '18'
+        cache: 'npm'
+        
+    - name: Install dependencies
+      run: npm ci
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)